Resources/Private/Form/StaticText.html

```diff
@@ -1,6 +1,7 @@
+{namespace form=TYPO3\Form\ViewHelpers}
```

Comment:
This PR is older than the namespace adjustment 😄 

But yea, I should have updated in the meantime. Will need to do that for my other old PRs as well